backend/src/helpers/mailtrap/mailtrap.ts

```diff
@@ -0,0 +1,41 @@
+import fs from 'fs';
+import path from 'path';
+
+import { ENV } from '@common/enums/app/app';
+import handlebars from 'handlebars';
+import nodemailer from 'nodemailer';
+
+const sendEmail = async (
+  email: string,
+  subject: string,
+  payload: unknown,
+  template: string,
+): Promise<void> => {
+  const transporter = nodemailer.createTransport({
+    host: ENV.MAILTRAP.EMAIL_HOST,
+    port: 587,
+    auth: {
+      user: ENV.MAILTRAP.EMAIL_USERNAME,
+      pass: ENV.MAILTRAP.EMAIL_PASSWORD,
+    },
+  });
```

Comment:
Can we share the `transport` between all the `sendEmail` invocations? Or should it be created for each email?